v2/pkg/options/systemtray.go

```diff
@@ -0,0 +1,22 @@
+package options
+
+import (
+	"github.com/wailsapp/wails/v2/pkg/menu"
+)
+
+// SystemTray contains options for the system tray
+type SystemTray struct {
+	LightModeIcon *SystemTrayIcon
+	DarkModeIcon  *SystemTrayIcon
+	Title         string
+	Tooltip       string
+	StartHidden   bool
```

Comment:
What does it mean to `StartHidden` on the `SystemTray`?

Reply:
It means it won't show it automatically on start up, but will rely on the developer calling `Show` when ready e.g. there may be the need to do some background tasks at application startup and you want to populate the menu based on the result of that.

Reply:
Understood, thanks for clearing that up.